packages/cohort/src/core.rs

```diff
@@ -288,8 +288,16 @@ impl Cohort {
 
         // Talos gave "go ahead"
 
+        // Check safepoint condition before installing ...
+
+        let safepoint = resp.safepoint.unwrap(); // this is safe for 'Committed'
```

Comment:
This will `panic` when the decision is aborted as `safepoint` will be `None`

Reply:
There is a "return" on line 286.